src/components/data-table/ha-data-table.ts

```diff
@@ -173,6 +173,10 @@ export class HaDataTable extends LitElement {
 
   private _sortColumns: SortableColumnContainer = {};
 
+  private _defaultSortColumn = "";
```

Comment:
```suggestion
  private _defaultSortColumn?: string;
```
This should be the same type as `sortColumn`.